bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/ui/wizard/CreateContractInputFromBusinessObjectWizardPage.java

```diff
@@ -153,27 +160,44 @@ private void createReminderText(final EMFDataBindingContext dbc, final Composite
             reminder.setText(Messages.reminderForStepMessage);
             autoGeneratedOperationButton.setText(Messages.autoGeneratedOperationStepButton);
             manuallyDefinedOperationButton.setText(Messages.manuallyDefinedOperationStepButton);
-            dbc.bindValue(actionObservable, PojoObservables.observeValue(this, "isAutogeneratedOperation"));
+            binding = dbc.bindValue(actionObservable, PojoObservables.observeValue(this, "isAutogeneratedOperation"));
         } else {
             reminder.setText(Messages.reminderForProcessMessage);
             autoGeneratedOperationButton.setText(Messages.autoGeneratedOperationProcessButton);
             manuallyDefinedOperationButton.setText(Messages.manuallyDefinedOperationProcessButton);
-            dbc.bindValue(actionObservable, PojoObservables.observeValue(this, "isAutogeneratedScript"));
+            binding = dbc.bindValue(actionObservable, PojoObservables.observeValue(this, "isAutogeneratedScript"));
         }
     }
 
+    public void checkMannuallyDefinedOperationButton() {
```

Comment:
ok it is a mistake in the name of the method. I mean to select the manuallyDefinedOperation button.